server/src/routes/kids/addKid.js

```diff
@@ -0,0 +1,19 @@
+import { StatusCodes } from 'http-status-codes';
+import Kid from '../../models/kid';
+
+const addKid = async (req, res) => {
+    const { name, dateOfBirth, height, weight, gender, avatar, diseases } = req.body;
+    const kid = new Kid({ name, dateOfBirth, height, weight, gender, avatar, diseases })
+
+    const isNameExists = await Kid.findOne({ name });
+    if (isNameExists) return res.status(StatusCodes.BAD_REQUEST).send({ message: 'Name already exist' });
+
+    try {
+        const newKid = await kid.save()
+        res.status(StatusCodes.CREATED).send(newKid)
+    } catch (err) {
+        res.status(StatusCodes.BAD_REQUEST).send({ message: err.message })
+    }      
+}
```

Comment:
This endpoint should have a little diffent logic. It should take the user that is making request from req.user (it will be provided in #40) and append it to kids that are already inside kids array. Right now the check for the uniqueness of the name is run across all kids created so two diffrent parent cannot name kid the same name 😄